execute/window_test.go

```diff
@@ -385,8 +409,8 @@ func TestWindow_GetOverlappingBounds(t *testing.T) {
 	}
 }
 
-func MustWindow(every, period, offset execute.Duration) execute.Window {
-	w, err := execute.NewWindow(every, period, offset)
+func MustWindow(every, period, offset execute.Duration, months bool) execute.Window {
+	w, err := execute.NewWindow(every, period, offset, months)
```

Comment:
@jlapacik the fix for that bug you saw includes awareness of nanoseconds vs months when creating the window.